contracts/modules/governance/Governance.sol

```diff
@@ -80,7 +78,6 @@ contract Governance is IGovernance, LegacyMasterAware {
 
   bool internal actionParamsInitialised;
   uint internal actionWaitingTime;
-  IMinimalCover internal cover;

```

Comment:
this was a newly added storage slot i presume and now we're rolling it back.

Reply:
It didn't made it to prod so there should be no issue. Let me know if you notice otherwise

Reply:
yes lgtm